mv_avs_dir.sh

```diff
@@ -52,32 +52,48 @@ usage() {
     output "usage: ${bold}$SCRIPT${normal} <source-directory> <destination-directory>"
 }
 
-rm_if_link(){
-    [ ! -L "$1" ] || rm "$1";
-}
 
-rm_simlink() {
-    cd $1
+## Operates files
+##   - Removes current similinks
+##   - Creates new simlinks
+## @param destination-directory
+## @param delimiter
+##
+op_files() {
+    ##
+    ## Removes simlinks
+    ##   and Creates pair of {simlink, original}
+    ##
+    dir=$1
+    cd "$dir"
+    declare -A array
+
     for i in *
     do
-        rm_if_link "$i"
+        f=$i
+        if  [ -L "$f" ] ; then
+            l=`readlink -f "$f"`
+            l=${l##*/}
+            rm "$f"
+            array[$l]=$f
```

Comment:
Here we can put the code block for creating symlinks into instead of tracking a symlink and a corresponding actual path in an associative array.

```shell
source_file="$dir/$l"
target_file="$dir/$f"
ln -fs "$source_file" "$target_file"
echo "[CREATED] $source_file > $target_file"
```